scripts/deploy.ts

```diff
@@ -1,15 +1,80 @@
-import { ethers } from "hardhat";
+import { Transaction } from 'ethers'
+import { namehash, keccak256, toUtf8Bytes, hexZeroPad } from 'ethers/lib/utils'
+import { ethers } from 'hardhat'
 
-async function main() {
-  const Greeter = await ethers.getContractFactory("Greeter");
-  const greeter = await Greeter.deploy("Hello, Hardhat!");
+const ETH_DOMAIN = 'eth'
+const MAIN_SUBDOMAIN = process.env.ENS_DOMAIN || 'fds'
+const FULL_SUBDOMAIN = `${MAIN_SUBDOMAIN}.${ETH_DOMAIN}`
+
+const INTERVAL = 1000
+const MAX_INTERVAL = 15 * INTERVAL
+
+function waitForTransactionMined(tx: Transaction): Promise<void> {
```

Comment:
move this function to the utils please

Reply:
There is some issue with linter configuration, it's throwing an error for all file imports. But it's a separate issue so it should be addressed on separate PR.